package.json

```diff
@@ -32,8 +32,6 @@
     "react-native-safe-area-context": "0.6.0",
     "react-native-screens": "2.0.0-alpha.12",
     "react-native-webview": "7.4.3",
-    "react-navigation": "^4.0.10",
-    "react-navigation-stack": "^2.0.0-alpha.42",
```

Comment:
Just making sure... we still get 2.0.0 alpha, or that's rolled into 5.0?

Reply:
`"@react-navigation/stack": "^5.0.0-alpha.46",` above, so I guess alpha.46 > alpha.42?